src/PaymentReceipt.tsx

```diff
@@ -17,6 +16,7 @@ import {
   ReceiptListItem,
   ReceiptButton,
 } from './components/receipts'
+import { PaymentReceiptItem } from '@oneblink/apps/dist/types/payments'
```

Comment:
This should be export as on paymentService as `paymentService.PaymentReceiptItem`. Ill fix apps now.